### PR TITLE
Basic model structure

### PR DESCRIPTION
**Pull request by steventux**
The basic data models for link checking.

### CODE REVIEW

File: db/migrate/20170403153216_create_links.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateLinks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :links do |t|
+      t.string :url
```
Comment (kevindew) [inline db/migrate/20170403153216_create_links.rb]: being pedantic this should probably be uri

File: db/migrate/20170403155048_create_jobs.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateJobs < ActiveRecord::Migration[5.0]
+  def change
+    create_table :jobs do |t|
+      t.datetime :last_checked_at
```
Comment (kevindew) [inline db/migrate/20170403155048_create_jobs.rb]: I think a job will only run once so we probably just need a completed_at timestamp here

File: db/migrate/20170403153216_create_links.rb (hunk 1 above)
Comment (steventux) [inline db/migrate/20170403153216_create_links.rb]: :+1: may as well get it right now.

File: db/migrate/20170403153216_create_links.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateLinks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :links do |t|
+      t.string :uri
```
Comment (steventux) [inline db/migrate/20170403153216_create_links.rb]: @kevindew I'm not sure if this restricts the length of the uri to a db-specific limit. Would `t.text` be better?

Comment (kevindew) [inline db/migrate/20170403153216_create_links.rb]: I don't think it does because Postgres varchar doesn't have a limit like MySQL does

**Pull request merged**